doc/content/verification_and_validation/val-2b.md

```diff
@@ -4,54 +4,72 @@
-
-Deuterium diffusion measurements in BeO were made by [!cite](fowler1977tritium). They found a wide range of results for diffusivity in BeO, depending on the physical form of the material, having measured it for single-crystal, sintered, and powdered BeO. This model uses one expression for the charging phase and another for the thermal desorption phase, believing that the surface film changed somewhat during the transfer between the two furnaces. For the charging phase diffusivity, the model uses 20 times that for the sintered BeO. Thermal expansion mismatches tend to open up cracks and channels in the oxide layer, so this seems a reasonable value. The same activation energy of 48.5 kJ/mol, is retained, however. For the thermal desorption phase, the diffusivity prefactor of the sintered material (7x10$^{-5}$ m$^2$/sec) and an activation energy of 223.7 kJ/mol (53.45 kcal/mol) are used. These values give good results and lie well within the scaller of Fowler's data. Exposure of the sample to air after heating should have made the oxide more like single crystal by healing the cracks that may have developed. Diffusivities and solubilities used in the simulation are listed below with T being the temperature in Kelvin:
-
-| Property of deuterium | Value for charging phase      | Value for desorption phase    |
-| --------------------- | ----------------------------- | ----------------------------- |
-| Diffusivity in Be     | 8.0x10$^{-9}$ exp(-4220/T)    | 8.0x10$^{-9}$ exp(-4220/T)    |
-| Diffusivity in BeO    | 1.40x10$^{-4}$ exp(-24408/T)  | 7x10$^{-5}$ exp(-27000/T)     |
-| Solubility in Be      | 7.156x10$^{27}$ exp(-11606/T) | 7.156x10$^{27}$ exp(-11606/T) |
-| Solubility in BeO     | 5.00x10$^{20}$ exp(9377.7/T)  | 5.00x10$^{20}$ exp(9377.7/T)  |
+Deuterium transport properties of the BeO are more challenging. First, it is not clear in which state the deuterium exists in the BeO. However, it has been observed [!cite](longhurst1990tritium) that an activation energy of -78 kJ/mol (exothermic reaction) is evident for tritium coming out of neutron irradiated beryllium in work done by D. L. Baldwin of Battelle Pacific Northwest Laboratory. The same value of energy has appeared in other results (can be inferred from Dr. Swansiger's work cited by [!cite](wilson1990beryllium) and by [!cite](causey1990tritium), among others), so one may be justified in using it. Concerning the solubility, measurements reported by [!cite](macaulay1992thermal) and in follow-up conversations indicate about 200 appm of D in BeO after exposure to 13.3 kPa fo D$_2$ at 773 K. That suggests a coefficient of only 1.88 x 10$^{18}$ d/m$^3$Pa$^{1/2}$. Since much of the deuterium in the oxide layer will get out during the cool-down process (and because it gives a good fit) the solubility coefficient is taken to be 5 $\times$ 10$^{20}$ d/m$^3$Pa$^{1/2}$.
```

Comment:
```suggestion
Deuterium transport properties of the BeO are more challenging. First, it is not clear in which state the deuterium exists in the BeO. However, it has been observed [!cite](longhurst1990tritium) that an activation energy of -78 kJ/mol (exothermic reaction) is evident for tritium coming out of neutron-irradiated beryllium in work done by D. L. Baldwin of Pacific Northwest Laboratory. The same value of energy has appeared in other results (can be inferred from Dr. Swansiger's work cited by [!cite](wilson1990beryllium) and by [!cite](causey1990tritium), among others), so one may be justified in using it. Concerning the solubility, measurements reported by [!cite](macaulay1992thermal) and in follow-up conversations indicate about 200 appm of D in BeO after exposure to 13.3 kPa of D$_2$ at 773 K. That suggests a coefficient of only 1.88 x 10$^{18}$ d/m$^3$Pa$^{1/2}$. Since much of the deuterium in the oxide layer will get out during the cool-down process (and because it gives a good fit), the solubility coefficient is taken to be 5 $\times$ 10$^{20}$ d/m$^3$Pa$^{1/2}$.
```

"Batelle Pacific Northwest Laboratory" is a tautology (then-PNL has only ever been managed by Batelle, who named it PNL).